### ISSUE TITLE
Add ability to remove private tabs from undo close history only after private browsing ends

### ISSUE THREAD

**Issue by Infocatcher** [Infocatcher/Private_Tab#112]
In addition for #6: we have secret _extensions.privateTab.rememberClosedPrivateTabs_ preference to not remove private tabs from undo close history, so it's nice to have ability to cleanup undo close history after closing of last private tab/window.

Private data will be saved to the disk (since we can't alter built-in session restore mechanism), but this may be appropriate for some users.
